packages/neptune-extension/src/common/components/checkout-modal/CheckoutModal.tsx

```diff
@@ -77,46 +84,61 @@ const CheckoutModal: React.FC<CheckoutModalProps> = ({
     onClose();
   }
 
-  const disabled = !projectMetaProps.valid || !notebookMetaProps.valid || !checkpointMetaProps.valid;
+  const disabled = projectMetaProps.status === 'error' || notebookMetaProps.status === 'error' || checkpointMetaProps.status === 'error';
```

Comment:
This one is wrong, the form is meant to be disabled when any value in invalid, not when it is invalid **and** finished loading.

Reply:
Ok, I'll fix that